### DISCUSSION TITLE
Format Error Messages

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3609] [Questions]
Fellow Csla developers,
how do you format error messages returned from broken business rules?
We have a few desktop applications (WPF/Winforms) and we have used 'MessageBox' to show error messages.
Has one of us tried grouping messages by property? or any other way?
Just to make it a little 'pretty' for users.

Regards

**Comment by rockfordlhotka** [comment]
It seems to me that someone (as part of [CslaContrib](https://github.com/marimerllc/cslacontrib)) once wrote a treeview control for broken rules - for WinForms I think.
